index.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Hello World</title>
+</head>
+<body>
+    <div class="hello">Hello World</div>
```

Comment:
> [OPTIONAL] I suggest you use the `h1` tag instead of the `div` for text. This enhances the accessibility and structure of your webpage, allowing screen readers and search engines to understand the hierarchy and importance of content. Don't worry you will learn about that😊